docs/guides/VTEX-Platform-Overview/developer-experience.md

```diff
@@ -0,0 +1,136 @@
+- [**Edition apps**](https://developers.vtex.com/docs/guides/vtex-io-documentation-edition-app): Bundles of settings, policies, backend, and frontend apps encapsulated into a single app.
+
+You can take advantage of our [VTEX IO apps](https://developers.vtex.com/docs/vtex-io-apps), which are pre-built solutions typically developed and maintained by VTEX that can be easily integrated into your store. These apps can be discovered, installed, and managed through the [VTEX App Store](#vtex-app-store).
+
+If you cannot find an app that meets your business needs, you can create customized solutions by [developing your own VTEX app](https://developers.vtex.com/docs/guides/vtex-io-documentation-developing-an-app) using the [VTEX IO](#app-development).
+
+We provide some ready-to-use [code templates](https://developers.vtex.com/docs/guides/code-samples) to simplify your development process when creating a VTEX IO app. Using one of them, you do not need to write the boilerplate code, ensuring development agility, quality and compliance with platform standards.
+
+>ℹ️ To learn about the relevant aspects to consider when developing VTEX IO apps, check out the [Before creating an app for your store](https://help.vtex.com/tracks/vtex-store-overview--eSDNk26pdvemF3XKM0nK9/7euXDZR5CCnVFSrXyczIhu#before-creating-an-app-for-your-store) article.
+
+##### **VTEX App Store**
+
+You can make your apps available to the VTEX ecosystem on the [VTEX App Store](https://developers.vtex.com/docs/guides/vtex-app-store), a marketplace for VTEX IO plug-and-play solutions.
+
+>ℹ️ To learn about the homologation requirements for the VTEX App Store, check out the [App Store Guidelines](https://developers.vtex.com/docs/guides/vtex-io-documentation-homologation-requirements-for-vtex-app-store).
+
+#### Deployment
+
+Once your new app has been developed, it must be deployed to the VTEX IO production workspace to make it publicly available.
+
+To [deploy a new app version](https://developers.vtex.com/docs/guides/vtex-io-documentation-making-your-new-app-version-publicly-available), follow these steps:
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/making-an-app-publicly-available.png)
+
+- [**Releasing a new app version**](https://developers.vtex.com/docs/guides/vtex-io-documentation-releasing-a-new-app-version): Marks the beginning by releasing the app using Git, updating the version in `manifest.json`, and documenting changes in CHANGELOG.md.
+- [**Publishing**](https://developers.vtex.com/docs/guides/vtex-io-documentation-publishing-an-app): The app version becomes a candidate version, ready for testing and validation in a production workspace.
+- [**Testing**](https://developers.vtex.com/docs/guides/ab-tests): Run [A/B tests](#ab-tests) and others, such as performance and security testing.
```

Comment:
If we don't have links with more information for performance and security testing, maybe this info should be removed as it might cause confusion